src/components/ExportRequestButton/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { forwardRef, memo, useState } from "react";
+import { styled, Button, ButtonProps } from "@mui/material";
+import { useSnackbar } from "notistack";
+import { ReactComponent as DownloadIcon } from "../../assets/icons/download_icon.svg";
+import StyledFormTooltip from "../StyledFormComponents/StyledTooltip";
+import { Status as FormStatus, useFormContext } from "../Contexts/FormContext";
+import { GenerateDocument } from "./pdf/Generate";
+import { downloadBlob, FormatDate } from "../../utils";
+
+const StyledTooltip = styled(StyledFormTooltip)({
+  marginLeft: "0 !important",
+  "& .MuiTooltip-tooltip": {
+    color: "#000000",
+  },
+});
+
+const StyledButton = styled(Button)({
+  "&.MuiButton-root": {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    minWidth: "128px",
+    padding: "10px",
+    fontFamily: "'Nunito', 'Rubik', sans-serif",
+    fontSize: "16px",
+    fontStyle: "normal",
+    lineHeight: "24px",
+    letterSpacing: "0.32px",
+    "& .MuiSvgIcon-root": {
+      fontSize: "20px",
+    },
+  },
```

Comment:
Related to this comment. turns out the button styling was redundant, as it's identical to the base theme. I've removed it 57586f2.

I also removed the color override on the SVG icon, and set it to inherit from the button coloring.

Enabled:

<img width="139" alt="Screenshot 2024-10-02 at 1 51 18 PM" src="https://github.com/user-attachments/assets/fd8e95f2-4086-40e8-a1b8-b9c5d4be861d">

Disabled:

<img width="139" alt="Screenshot 2024-10-02 at 1 51 30 PM" src="https://github.com/user-attachments/assets/5b8a11a0-8d10-4701-96b9-b96a7283b406">

The icon still looks somewhat lighter than the text color, but I think that's just because of the width of the lines in the SVG? Lmk if you think further adjustments are needed.